### PR TITLE
feat: add brew command 'postinstall'

### PR DESCRIPTION
**Pull request by ShawnCockburn**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

it fixes a missing `brew` command: `brew postinstall`.

**What is the current behavior? (You can also link to an open issue here)**

autocomplete is missing this command

**What is the new behavior (if this is a feature change)?**

`brew postinstall` is shown in autocomplete 

**Additional info:**

screenshot of missing command `--help`

### CODE REVIEW

File: src/brew.ts [hunk 1]
```diff
@@ -741,6 +741,31 @@ const completionSpec: Fig.Spec = {
       name: "config",
       description: "Show Homebrew and system configuration info",
     },
+    {
+      name: "postinstall",
+      description: "Postinstall <formula>",
```
Comment (fedeci) [inline src/brew.ts:746]:
```suggestion
      description: "Rerun the post install step for formula",
```

File: src/brew.ts [hunk 2]
```diff
@@ -741,6 +741,31 @@ const completionSpec: Fig.Spec = {
...
+        },
+
+        {
+          name: ["-q", "--quiet"],
+          description: "Make some output more quiet",
+        },
+        { name: ["-h", "--help"], description: "Show this message" },
+      ],
+      args: {
+        isVariadic: true,
```
Comment (fedeci) [inline src/brew.ts:764]: I am not sure if this is variadic, the description seems to refer to a single spec?

**Review by fedeci — CHANGES_REQUESTED** [review]

File: src/brew.ts (hunk 2 above)
Comment (fedeci) [reply src/brew.ts:764]: Sorry, I meant single formula, that's correct, it is variadic!
Concerning the description I get the output I suggested, are you on the latest version of brew?

Comment (ShawnCockburn) [reply src/brew.ts:764]: > Sorry, I meant single formula, that's correct, it is variadic!
> Concerning the description I get the output I suggested, are you on the latest version of brew?

[I am silly](https://github.com/withfig/autocomplete/pull/862#issue-1110415874) I must have not copied it, my bad.

thanks :)

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**